daemons/gptp/common/avbts_clock.hpp

```diff
@@ -92,6 +92,13 @@ class IEEE1588Clock {
 	bool grandmaster_is_boundary_clock;
 	uint8_t time_source;
 
+  ExtPortConfig external_port_configuration; // IEEE 1588 defaultDS.externalPortConfiguration
```

Comment:
Is there an existing datatype that you can use to store the state instead of creating a new datatype?